vasquez_a07b.js

```diff
@@ -0,0 +1,127 @@
+/*
+Bryan Vasquez
+Vasquez_a07b.js
+INFO 2124
+Thoendel
+02/02/2020
+*/
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+
+//Step 1
+buildFoodsList (badFoods, "bread", "beer","pizza", "meat", "candy"); //Declaring builFoodsList array with 5 items 
+
+standardInput.on('data', function (text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    //Step 4 Begin
+    clearScreen();
+    while(true) {                                                  //While loop to repeat the user input for adding more items.
+    if (text === "Q"||"q"){                                        //If enter Q or q the program will end.
+        clearScreen();
+        console.log("Bye!");
+        process.exit();
+    }
+    else if (text === "V"||"v"){                                   //This will show the item list
+        clearScreen();
+        displayList(groceryItems);
+        console.log(displayList());
+        console.log(getPrompt(groceryItems));
+    }
+    else if (text === checkItem(badFoods)){                        //This will avoid the bad food for our health

```

Comment:
the text won't === true or false. if you write it the way I've suggested it should be something like:
else if (checkkItem(badFoods)){//the food is bad}
else {//the food is good and should be added}